ROLES.md

```diff
@@ -276,6 +276,30 @@ The following apply to the area / component for which one would be a lead.
 
   - Mentoring and guiding approvers, members, and new contributors.
 
+
+## Inactivity
+It is important for contributors to be and stay active to set an example and show commitment to the project. Inactivity is harmful to the project as it may lead to unexpected delays, contributor attrition, and a lost of trust in the project.
+
+* Inactivity is measured by:
+    * Periods of no contributions for longer than 12 months
+    * Periods of no communication for longer than 12 months
+* Consequences of being inactive include:
+    * Involuntary removal or demotion
```

Comment:
I'm a bit torn on the idea of demotion... given that a member is defined as "an established Contributor who regularly participates in the project", I could see an argument for just straight dropping folks who haven't contributed in a year.

Reply:
I like the idea of just removing inactive folks instead of demoting as well. Demoting, for me at least, would be a bit offensive to me. I would rather just be silently removed.